spec/unit/spark_api/models/saved_search_spec.rb

```diff
@@ -99,6 +99,27 @@
       resource.ContactIds.size.should eq(0)
     end
 
+    describe "listings" do
+
+      it "should return the searches listings" do
+        stub_api_get("/#{subject.class.element_name}/#{id}", 'saved_searches/get.json')
+        stub_api_get("/listings", 'listings/multiple.json', 
+          {:_filter => "City Eq 'Moorhead' And MlsStatus Eq 'Active' And PropertyType Eq 'A'"})
+        listings = subject.class.find(id).listings
+        listings.should be_an(Array)
+        listings[0].should be_a(Listing)
+      end
+      
+      it "should include the permissive parameter for provided searches" do
```

Comment:
I'd recommend a test that ensures `:RequestMode => 'permissive'` is not included for non-provided searches.